features/urlpattern.yml

```
@@ -0,0 +1,19 @@
+name: URLPattern
+description: The `URLPattern` API creates patterns that can be matched against URLs or URL components.
+spec: https://urlpattern.spec.whatwg.org/
+status:
+  compute_from: api.URLPattern
```

For `compute_from` I would recommend the entrypoint to the feature, the thing you have to use in order to use the feature at all. In this case it's the `URLPattern` constructor:

```suggestion
  compute_from: api.URLPattern.URLPattern
```

I see that's also missing from `compat_features`. My approach is to use `npm run traverse | grep -i urlpattern` or similar, and start from that list.

---

Yeah, I've used this here, but didn't keep the constructor. For some reason, I was under the impression that we had an old guideline about not including constructors. But, thinking about it again, I think that was only about events.

---

Ah, indeed, because so many event constructors were added long after the interfaces themselves.

Now that features are "living" however, we could include event constructors as well.